edge/pkg/edged/edged.go

```diff
@@ -698,6 +699,13 @@ func (e *edged) initializeModules() error {
 		return err
 	}
 
+	// start the CPU manager in the clcm
+	err = e.clcm.StartCPUManager(e.GetActivePods, edgedutil.NewSourcesReady(e.isInitPodReady), e.statusManager, e.runtimeService)
```

Comment:
If we don't add `StartCPUManager` here, can the ci test pass? If ci can pass, maybe we should add the `feature` in another PR :)

Reply:
If we don't add this function here, the CPU manager won't start, and one of its fields remains nil, which will lead to edgecore crashing. 

That field was not nil in previous k8s versions, but its implementation changed so that the field remains nil until the start function is called.

Reply:
Ok, and can we move the `StartCPUManager` to Start func instead of `newedged`?

Reply:
fixed.